docs/reference/workflows/monobehaviour/worker-connectors.md

```diff
@@ -0,0 +1,140 @@
+<%(TOC)%>
+
+# MonoBehaviours: Worker connectors
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+  * [Connection flows]({{urlRoot}}/reference/concepts/connection-flows)
+")%>
+
+To expose the [`Worker`]({{urlRoot}}/api/core/worker) class in the MonoBehaviour workflow, the GDK provide an abstract [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector) MonoBehaviour.
```

Comment:
I'd explain what the worker connector is first before mentioning you can access the Worker through it (flip paragraphs + smooth out to flow)

Reply:
I guess the intention is you've read about the `Worker` class (noted in the box at the top) - now lets segue into the `WorkerConnector` 🤔